### PR TITLE
download

### PR DESCRIPTION
**Pull request by chuanchyone1**
image

### CODE REVIEW

File: chrome/downloads/sampleFile.jpeg
Comment (danielgalvis98) [inline chrome/downloads/sampleFile.jpeg:1]: Delete this file

File: cypress/downloads/sampleFile.jpeg
Comment (danielgalvis98) [inline cypress/downloads/sampleFile.jpeg:1]: Please add "cypress/downloads/" to the gitignore and delete the file

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// import "cypress-file-upload";
 import "./commands";
+require("cypress-downloadfile/lib/downloadFileCommand");
```
Comment (danielgalvis98) [inline cypress/support/commands.ts]: Why this require?

File: cypress/support/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+module.exports = () => {
```
Comment (danielgalvis98) [inline cypress/support/index.ts]: Why this change?

File: cypress/support/commands.ts (hunk 1 above)
Comment (chuanchyone1) [reply cypress/support/commands.ts]: This line has been deleted

File: chrome/downloads/sampleFile.jpeg
Comment (chuanchyone1) [reply chrome/downloads/sampleFile.jpeg:1]: This file has been deleted

File: cypress/support/index.ts (hunk 2 above)
Comment (chuanchyone1) [reply cypress/support/index.ts]: This has been deleted

File: cypress/downloads/sampleFile.jpeg
Comment (chuanchyone1) [reply cypress/downloads/sampleFile.jpeg:1]: made

**Review by danielgalvis98 — APPROVED** [review]
